resources/views/layouts/app.blade.php

```diff
@@ -139,6 +139,8 @@
 
                   @if (\Auth::user()->isHeadMaster())
                     <li><a  href="{{ route('dashboard') }}" class="nav-item is-tab"><span class="icon"><i class="fa fa-bar-chart"></i></span> Dashboard</a></li>
+                    <li><a href="{{ route('news') }}" class="nav-item is-tab"><span class="icon"><i class="fa fa-newspaper-o"></i></span>News</a></li>
```

Comment:
Question: you are going directly to news, but don't you need to go to dashboard first?

Reply:
no we dont need to go to the dashboard first because the news page in the dashboard has a route that does go to the newspage. i did test it it works fine

Reply:
The dashboard page is the framework see view/layout/dashboard around the pages in the dashboard, so if you call the news route you get in the dashboard on the newspage